File: 프로그래머스/0/120846. 합성수 찾기/합성수 찾기.cs
using System;
using System.Collections.Generic;

public class Solution 
{
    public int solution(int n) 
    {
        List<int> composite = new List<int>();
        for(int i=1; i<=n; i++)
        {
            int count = 0;
            for(int j=1; j<=i; j++)
            {
                if(i%j == 0)
                {
                    count++;
                }
            }
            if(count >= 3)
            {
                composite.Add(i);
            }
        }
        return composite.Count;
    }
}

File: 프로그래머스/0/120846. 합성수 찾기/합성수 찾기.cpp
#include <string>
#include <vector>

using namespace std;

int result = 0;
int solution(int n) 
{
    for(int i=1; i<=n; i++)
    {
        int c = 1;
        int count = 0;
        while(c <= i)
        {
            if(i%c == 0)
            {
                count++;
            }
            c++;
        }
        if(count >= 3)
        {
            result++;
        }
    }
    return result;
}